packages/middleware-flexible-checksums/src/getChecksumAlgorithmForRequest.ts

```diff
@@ -11,6 +16,11 @@ export interface GetChecksumAlgorithmForRequestOptions {
    * Defines a top-level operation input member that is used to configure request checksum behavior.
    */
   requestAlgorithmMember?: string;
+
+  /**
+   * Determines when a checksum will be calculated for request payloads
+   */
+  requestChecksumCalculation: string;
```

Comment:
are the values an enum?

Reply:
Fixed in [`08bec3a` (#6484)](https://github.com/aws/aws-sdk-js-v3/pull/6484/commits/08bec3a5b2867dc97f5e649fa24f2fa8ed2b88c6)